### PR TITLE
Fix all tests

### PR DESCRIPTION
**Pull request by maxsmith271346**
## Description

- Fixed all failing Cucumber steps and scenarios
- Fixed all failing RSpec tests
- Removed Code Climate Coverage integration due to bug
- TODO: Add a Simplecov action to the workflow

## How has this been tested?

All Cucumber and RSpec tests were run

## Screenshots (if appropriate):
image

## Checklist:

- [x ] Zero `rubocop` violations
- [ ] My code receives an "A" when `rubycritic` is run
- [x ] >90% test coverage of my changes
- [x ] All new and existing tests passed

### CODE REVIEW

File: features/step_definitions/update_tests_file_steps.rb [hunk 1]
```diff
@@ -1,29 +1,118 @@
...
+        When with the target "#{string}"
+  )
+end
+
+When('I fill in the test block with {string}') do |string|
+  fill_in 'Actual test', with: string
+end
+
+Then('I should see a success message') do
+  expect(page).to have_content('Test was successfully')
```
Comment (masarkar-tamu) [inline features/step_definitions/update_tests_file_steps.rb:65]: Shouldn't the message say "Test was successfully created"

File: features/step_definitions/create_assignment_steps.rb [hunk 2]
```diff
@@ -16,12 +16,11 @@
 After do
   RSpec::Mocks.teardown
   WebMock.disable!
-  FileUtils.rm_rf('assignment-repos')
+  # FileUtils.rm_rf('assignment-repos')
```
Comment (masarkar-tamu) [inline features/step_definitions/create_assignment_steps.rb:19]: Would you be able to explain the reasoning behind commenting this line?

File: features/step_definitions/update_tests_file_steps.rb (hunk 1 above)
Comment (maxsmith271346) [reply features/step_definitions/update_tests_file_steps.rb:65]: There are multiple messages that say `Test was successfully <action>". I'll see if I can replace with a regex to make it more clear.

File: features/step_definitions/create_assignment_steps.rb (hunk 2 above)
Comment (maxsmith271346) [reply features/step_definitions/create_assignment_steps.rb:19]: Fixed

File: spec/controllers/user_controller_spec.rb [hunk 3]
```diff
@@ -1,94 +1,92 @@
...
+
+  before do
+    allow(controller).to receive(:session).and_return({ github_token: 'fake_token' })
+    @mock_client = instance_double(Octokit::Client)
+    allow(Octokit::Client).to receive(:new).and_return(@mock_client)
+  end
+
+  describe "when all the users in the organization are displayed" do
+    it "assigns users" do
+      user = User.create!(name: 'Test User')
```
[the remark this answers is not in the document]
Comment (maxsmith271346) [reply spec/controllers/user_controller_spec.rb:16]: Fixed

**Review by masarkar-tamu — APPROVED** [review]

**Pull request merged**